draft-ietf-tls-tls13.md

```diff
@@ -2612,6 +2625,11 @@ facilitates implementation as described below). Servers MUST check
 that it is the last extension and otherwise fail the handshake with an
 "illegal_parameter" alert.
 
+When the syntax of this extension is correct but the server cannot
+verify at least one binder because of some configuration or state
+changes, the server SHOULD fall back to a full 1-RTT handshake if a
+"key_share" extension is supplied.
```

Comment:
If the binder is present and the server believes that it has a key, then it MUST fail the handshake (in my view).  That's certainly what NSS does.  If the binder is present, but the server doesn't have a key, then it doesn't have a key and can't resume.